diff --git a/spacelift.pkr.hcl b/spacelift.pkr.hcl
index d43253e..a1d00bb 100644
--- a/spacelift.pkr.hcl
+++ b/spacelift.pkr.hcl
@@ -1,41 +1,35 @@
 variable "ami_regions" {
   type = list(string)
   default = [
+    "us-east-1",
-    "ap-southeast-1",
-    "eu-west-1",
-    "us-west-2"
   ]
 }
-
 variable "base_ami" {
   type = string
+  default = "ami-0e999cbd62129e3b1"
 }
-
 source "amazon-ebs" "spacelift" {
   source_ami = var.base_ami
-
+  encrypt_boot  = true
   instance_type = "t2.micro"
   ssh_username  = "ec2-user"
-
-  ami_name    = "spacelift-{{timestamp}}"
+  ami_name    = "spacelift"
   ami_regions = var.ami_regions
-  ami_groups  = ["all"]
-
+  #ami_groups  = ["all"]
   tags = {
     Name    = "Spacelift AMI"
     Purpose = "Spacelift"
     BaseAMI = var.base_ami
   }
 }
-
 build {
   sources = ["source.amazon-ebs.spacelift"]
-
   provisioner "file" {
     source      = "configs/"
     destination = "/tmp"
   }
-
   provisioner "shell" {
     scripts = [
       "scripts/01-data-directories.sh",
